src/zrc/tests/test_notifications_send.py

```diff
@@ -31,10 +31,26 @@
 @override_settings(
     LINK_FETCHER="vng_api_common.mocks.link_fetcher_200", NOTIFICATIONS_DISABLED=False
 )
-class SendNotifTestCase(JWTAuthMixin, APITestCase):
+class SendNotifTestCase(JWTAuthMixin, APITransactionTestCase):
```

Comment:
with the use of `capture_on_commit_callbacks` this can be reverted back to normal APITestCase @stevenbal